mloop/learners.py

```diff
@@ -851,6 +865,8 @@ def __init__(self,
                                   'population_size':self.population_size,
                                   'num_population_members':self.num_population_members,
                                   'restart_tolerance':self.restart_tolerance,
+                                  'restart_to_best':self.restart_to_best,
+                                  'lifetime':self.lifetime,
```

Comment:
I meant to write this comment on my PR review, but looks like it didn't go through. It would be good to save the setting for `self.elite` here as well.